devstack/Readme-NFP-install.txt

```diff
@@ -26,7 +26,7 @@ Fresh Installation Steps:
 
     * Enterprise Mode Configuration:
          # Devstack installation in enterprise mode
-             Example: [[ $ENABLE_NFP = True ]] && DEVSTACK_MODE=enterprise
```

Comment:
We need to remove enterprise related instructions as we are going to submit this as part of advanced mode.